cluster/sync.sh

```diff
@@ -27,6 +27,9 @@ make push
 make cluster-clean
 make deploy
 
+# Ensure the project network-polices are valid by installing an additional deny-all network-policy affecting the project namespace
+./hack/install-deny-all-net-pol.sh
```

Comment:
should be after pods_ready_wait, and then you dont need 

https://github.com/kubevirt/kubesecondarydns/pull/99/files?diff=unified&w=0#diff-347769588be178babe855e27568abd38df2e51608ff03dea81caf1175afb3fc4R21

Reply:
The namespace existence check is necessary in case the ns was deleted by mistake during development iteration.

It create the network-policy before waiting for pods just in case the network-policy took more time to take place, so that in this scenario we have more time before tests will be executed, and better verification pods are ready when network-policies are installed.
Make sense?

Reply:
no sorry, this shouldnt happen, we dont need to protect against such mistakes
it is never ending story, and we dont have such things atm, and good that we arent

Reply:
According to k8s docs, network-policies will take place eventually, by the network-plugin.

I would install network-policies in cluster-up and get done with it because its part of the setup, but naturally we dont have ksd namespace yet. 
This is why its installed as part of cluster-sync, its the closet to cluster-up in the sense that it can faciliate configure the env just before deploying the project or run tests.

Maybe now that the script is simplified a bit it following https://github.com/kubevirt/kubesecondarydns/pull/99#discussion_r2100467486 it will make more sense, PTAL